egs/iam/s5/local/prepare_data.sh

```diff
@@ -22,7 +22,7 @@ if [ -f path.sh ]; then . ./path.sh; fi
 . parse_options.sh || exit 1;
 
 # download dir
-download_dir=data/download
+download_dir=/export/corpora5/handwriting_ocr/IAM
```

Comment:
Ashish, if you change it here, it will be almost hardcoded for CLSP grid.
As I said, it's better to leave it as is (i.e. `data/download`) and instead
change it when calling this script from run.sh (i.e. pass it as an argument)